instance-flavor-check

```diff
@@ -17,6 +17,12 @@
 # You should have received a copy of the GNU General Public License along with
 # instance-billing-flavor-check. If not, see <http://www.gnu.org/licenses/>.
 
+import os
+import sys
+
+if os.geteuid():
+    sys.exit('You must be root')
```

Comment:
This really only applies to Azure because we read the billing tag :( but we have to live with it; bummer. Anyway could you please add a comment here. 
```
# Need to be root in Azure because we read from a specific area of the disc
```

Reply:
Yes,
`# Need to be root in Azure because we read from a specific area of the disc` 
is one reason the other is that we write to `/var/log/<log_name>` and it crashes without being sudo

will add the comment